src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -101,6 +102,59 @@ We recommend tracking the following metrics:
+
+`SHOW EXPERIMENTAL_FINGERPRINT` allows you to verify that the data transmission and ingestion is working as expected while a replication stream is running. Any checksum mismatch likely represents corruption or a bug in CockroachDB. Should you encounter such a mismatch, contact [Support](https://support.cockroachlabs.com/hc/en-us).
+{{site.data.alerts.end}}
+
+To verify that the data at a certain point in time is correct on the standby cluster, you can use the current replicated time from the replication job information to run a point-in-time fingerprint on both the primary and standby clusters. This will verify that the transmission and ingestion of the data on the standby cluster, at that point in time, is correct.
+
+1. Retrieve the current replicated time of the replication job on the standby cluster with [`SHOW VIRTUAL CLUSTER`]({% link {{ page.version.version }}/show-virtual-cluster.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT replicated_time FROM [SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS];
+    ~~~
+    ~~~
+         replicated_time
+    ----------------------------
+    2024-01-09 16:15:45.291575+00
+    (1 row)
+    ~~~
+
+    For detail on connecting to the standby cluster, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connect-to-the-standby-cluster-system-interface).
+
+1. From the **primary cluster's system interface**, specify a timestamp at or lower than the current `replicated_time` to verify the data:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT * FROM [SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER application] AS OF SYSTEM TIME '2024-01-09 16:15:45.291575+00';
+    ~~~
+    ~~~
+    tenant_name |             end_ts             |     fingerprint
+    ------------+--------------------------------+----------------------
+    application | 1704816945291575000.0000000000 | 2646132238164576487
+    (1 row)
+    ~~~
+
+    For detail on connecting to the primary cluster, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connect-to-the-primary-cluster-system-interface).
+
+1. From the **standby cluster's system interface**, specify a timestamp at or lower than the current `replicated_time` to verify the data:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT * FROM [SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER standbyapplication] AS OF SYSTEM TIME '2024-01-09 16:15:45.291575+00';
+    ~~~
+    ~~~
+        tenant_name     |             end_ts             |     fingerprint
+    --------------------+--------------------------------+----------------------
+    standbyapplication  | 1704816945291575000.0000000000 | 2646132238164576487
+    (1 row)
+    ~~~
+
```

Comment:
```suggestion
1. Compare the fingerprints of the primary and standby clusters to verify the data. The same value for the fingerprints indicates the data is correct. A mismatch in fingerprint values indicates corruption in the data or a bug in CockroachDB. 
```

Reply:
Added this, though not the second sentence — this is covered in the note at the top of the section, which also includes the call to contact us. This is probably important to keep together for a couple of reasons.